src/components/SourceSearch.js

```diff
@@ -58,13 +58,17 @@ const Search = React.createClass({
   componentWillUnmount() {
     const shortcuts = this.context.shortcuts;
     shortcuts.off(`CmdOrCtrl+${L10N.getStr("sources.search.key")}`,
-      this.toggle);
+     this.toggle);
+    shortcuts.off(`CmdOrCtrl+${L10N.getStr("sources.searchAlt.key")}`,
+     this.toggle);
```

Comment:
I implemented your suggestions yesterday and pushed to my branch, but the changes doesn't seem to show up here. I remember seeing the changes show up yesterday though. (First time open source contributor, not sure if I'm missing something)